fusor/fusor.py

```diff
@@ -1,10 +1,11 @@
 """Module for modifying fusion objects."""
+import re
 from typing import Dict, List, Optional, Tuple
 from urllib.parse import quote
 
 from biocommons.seqrepo import SeqRepo
 from bioutils.accessions import coerce_namespace
-from cool_seq_tool import CoolSeqTool
+from cool_seq_tool.routers import CoolSeqTool
 from cool_seq_tool.schemas import ResidueMode
 from ga4gh.core import ga4gh_identify
 from ga4gh.vrs import models
```

Comment:
Change `from pydantic.error_wrappers import ValidationError` to `from pydantic import ValidationError`